Templates/Ros2ProjectTemplate/README.md

```diff
@@ -1,13 +1,17 @@
 # Robotic Simulation Project Template for ROS 2
 
-Welcome to Robotic Simulation template for ROS 2 projects! 
-This project is a template designed to help you quickly start simulating robots in Open 3D Engine (O3DE) with Robot Operating System (ROS) 2.
+Welcome to Robotic Simulation template for ROS 2 projects!
+It is designed to help you quickly start simulating robots in Open 3D Engine (O3DE) with Robot Operating System (ROS) 2.
 
-An example ROS2 navigation stack launch file is bundled with the template.
+This template has the following characteristics:
+- It is the simplest and using the least resources among templates.
+- Comes with Husarion ROSBot XL model, equipped with lidar and a differential drive.
+- The interior scene does not look photorealistic but is quite simple.
```

Comment:
Suggestion;
`- The interior scene is simple and lightweight, however does not look photorealistic.`